File: examples/src/Ampel/Ampel.py
import sys
import ftrobopy
import time

ftrob = ftrobopy.ftrobopy('auto')
red = ftrob.output(1)
yellow = ftrob.output(2)
green = ftrob.output(3)
while True:
    red.setLevel(512)
    time.sleep(3)
    yellow.setLevel(512)
    time.sleep(1)
    red.setLevel(0)
    yellow.setLevel(0)
    green.setLevel(512)
    time.sleep(3)
    green.setLevel(0)
    yellow.setLevel(512)
    time.sleep(1)
    yellow.setLevel(0)

File: examples/src/Ampel/Ampel.cpp
#include <unistd.h>         // for sleep()

#include "KeLibTxtDl.h"     // TXT Lib
#include "FtShmem.h"        // TXT Transfer Area

int main(void) {
	FISH_X1_TRANSFER *pTArea;
	if (StartTxtDownloadProg() == KELIB_ERROR_NONE) {
		pTArea = GetKeLibTransferAreaMainAddress();
		if (pTArea) {	
			while(true){
				pTArea->ftX1out.duty[0] = 512;			//red
				sleep(3);
				pTArea->ftX1out.duty[1] = 512;			//yellow
				sleep(1);
				pTArea->ftX1out.duty[0] = 0;			//red
				pTArea->ftX1out.duty[1] = 0;			//yellow
				pTArea->ftX1out.duty[2] = 512;			//green
				sleep(3);
				pTArea->ftX1out.duty[2] = 0;			//green
				pTArea->ftX1out.duty[1] = 512;			//yellow
				sleep(1);
				pTArea->ftX1out.duty[1] = 0;			//yellow
			}			
		}
		StopTxtDownloadProg();
	}
	return 0;
}
